.dotnet/api/OpenAI.netstandard2.0.cs

```diff
@@ -1738,26 +1373,70 @@ public class FineTuningClient {
         public virtual ClientResult GetJobs(string after, int? limit, RequestOptions options);
         public virtual Task<ClientResult> GetJobsAsync(string after, int? limit, RequestOptions options);
     }
+    public class FineTuningJob {
+        public DateTimeOffset CreatedAt { get; }
+        public FineTuningJobError Error { get; }
+        public DateTimeOffset? EstimatedFinish { get; }
+        public string FineTunedModel { get; }
+        public DateTimeOffset? FinishedAt { get; }
+        public FineTuningJobHyperparameters? Hyperparameters { get; set; }
+        public string Id { get; }
+        public string Model { get; }
+        public string Object { get; }
+        public string OrganizationId { get; }
+        public IReadOnlyList<string> ResultFiles { get; }
+        public int Seed { get; }
+        public FineTuningJobStatus Status { get; }
+        public int? TrainedTokens { get; }
+        public string TrainingFile { get; }
+        public string ValidationFile { get; }
+    }
+    public class FineTuningJobError {
+        public string Code { get; }
+        public string Message { get; }
+        public string Param { get; }
+    }
+    public partial struct FineTuningJobHyperparameters {
```

Comment:
Reading this closely, it looks like there's a significant difference between the request/input representation of Hyperparameters and the response/output representation -- the request side is a lot more complicated!

I'd suggest:
- Create a new `FineTuningHyperparameterOptions` class that uses `[CodeGenModel("CreateFineTuningJobRequestHyperparameters")] as its basis -- I believe that's the generated source material for the request side (and why things likely appeared to be missing)
- Keep the existing type for the response model (which will still need to solve the "int or string" problem but not have the other fields)
- We can consider (later) if we want the response type to more transparently differentiated by name. e.g. "FineTuningJobHyperparameterInfo" or similar